src/javascript/app/pages/trade/charts/highchart.js

```diff
@@ -488,20 +488,16 @@ const Highchart = (() => {
     // calculate where to display the maximum value of the x-axis of the chart for line chart
     const getMaxHistory = (history_times) => {
         let end = end_time;
-        if (sell_spot_time && (sell_time || sell_spot_time) < end_time) {
-            end = sell_spot_time;
+        if (sell_time && sell_time < end_time) {
+            end = sell_time;
```

Comment:
our line is drawn at `sell_time`, we should end on the tick after `sell_time` so the line is always visible